src/main/java/com/amazonaws/encryptionsdk/keyrings/RawRsaKeyringBuilder.java

```diff
@@ -97,6 +97,31 @@ public RawRsaKeyringBuilder wrappingAlgorithm(String wrappingAlgorithm) {
      * @return The {@link Keyring} instance
      */
     public Keyring build() {
-        return new RawRsaKeyring(keyNamespace, keyName, publicKey, privateKey, wrappingAlgorithm);
+        return new RawRsaKeyring(keyNamespace, keyName, publicKey, privateKey, paddingScheme);
+    }
+
+    public enum RsaPaddingScheme {
```

Comment:
NitPick: For ease of use, calculate the correct OAEPParameterSpec instances for each of these (they are immutable) and just pull them from the enum.

Reply:
I'll hold off on this until Master Keys are deleted